advanced/Scripts/utils.sh

```diff
@@ -15,7 +15,10 @@
 #  - New functions must have a test added for them in test/test_any_utils.py
 
 #######################
-# Takes three arguments key, value, and file.
+# Takes either
+#   - Three arguments: key, value, and file.
+#   - Two arguments: key, and file
+#
 # Checks the target file for the existence of the key
 #   - If it exists, it changes the value
 #   - If it does not exist, it adds the value
```

Comment:
Thank you for function comments!!

Reply:
I may be talking nonsense, but wouldn't it be easier to remember, more readable (and simpler to test) if we used a different parameters order? 

Since the filename and key must always be present, wouldn't it be better if it were this way?

```sh
addOrEditKeyValuePair {filename} {key} {value (optional)}

addOrEditKeyValuePair "/etc/pihole/setupVars.conf" "BLOCKING_ENABLED" "true"
addOrEditKeyValuePair "/etc/pihole/setupVars.conf" "KEY_WITH_NO_VALUE"
```

Reply:
I agree with @rdwebdesign, changing the order makes a lot of sense to me.